server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_2__add_object_search.sql

```diff
@@ -0,0 +1,29 @@
+-- Record all database objects such as table, view, function, procedure, package,  package_body, trigger, type, sequence, synonym and public synonym
+CREATE TABLE IF NOT EXISTS `connect_database_object` (
```

Comment:
let's discuss about module naming.

all table name start with `connect_ `  prefix belong to 'data source' module (aka 'connection' module before odc 4.2.0),  we keep `connect_` prefix for compatibility.

'database' concept may belong to 'data source' module due user must select a database to open an database session.

now, we got more database meta information to store in odc metadb.

includes  table/view/procedure objects, includes table column/view column/index columns/ ...
these object is not directly related with database session.

what if we involve a new module for database shema management, such as 'database_schema'.
then table name here may 'database_schema_object', 'database_schema_column'

Reply:
Yes, it would make more sense to have the database metadata management as a separate module. I have modified it as you suggested.